site/docs/voice/webhooks/bridgeTargetComplete.mdx

```diff
@@ -18,6 +18,8 @@ import TabItem from '@theme/TabItem';
 
 If the originating call leaves the [`<Bridge>`](/docs/voice/bxml/bridge), then this callback is sent to the `bridgeTargetCompleteUrl`, and the BXML returned in it is executed on the target call.
 
+Note: if this webhook is sent, the [Bridge Complete](/docs/voice/webhooks/bridgeComplete) webhook is NOT sent.
```

Comment:
```suggestion
Note: if this webhook is sent, the [Bridge Complete](/docs/voice/webhooks/bridgeComplete) webhook will not be sent.
```

Maybe?

Reply:
This applies to the other places, too

Reply:
I think emphasis of potential gotchas shouldn't necessarily be avoided.